### PR TITLE
update TFP code

### PR DESCRIPTION
**Pull request by aloctavodia**
* Chapter 6 (Times Series) keeps crashing on my machine, so I was not able to rerun the entire notebook

* At some point the first cell of the notebook was changed to "markdown", instead of "code", I turned back to code and removed the message.

* The environment file is also updated

### CODE REVIEW

**Comment by canyon289** [comment]
Since its crashing does @junpenglao need to run or verify chapter 6?

**Comment by aloctavodia** [comment]
That would be great. Alternatively, I can remove that notebook from this PR. Or do something else...

**Comment by canyon289** [comment]
IMO just add a note that says "this notebook is note completed" at the top and we can merge

File: notebooks_updated/chp_06.md [hunk 1]
```diff
@@ -0,0 +1,1803 @@
...
+import datetime
+print(f"Last Run {datetime.datetime.now()}")
+```
+
+```python
+az.style.use("arviz-grayscale")
+plt.rcParams["figure.dpi"] = 300
+```
+
+## Time Series Analysis as a Regression Problem
```
Comment (canyon289) [inline notebooks_updated/chp_06.md:40]: Note: This notebook is failing execution for us at the moment. Use it as a reference. We will have it updated shortly

**Pull request merged**